release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,43 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-enhanced-disk-provisioning"]
+==== Enhanced disk provisioning for LUKS and RAID devices
+
+Disk tooling enhancements now support complex root devices in bare metal deployments. For example, software RAID improvements now allow for more robust nodes that can still reboot in the event of a drive failure. And support for multiple LUKS root devices means that you now have added capabilities such as using the Local Storage Operator in file mode, having your non-rootfs data encrypted at rest, and use of third-party software that requires local persistent storage.
+
+{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+
+* Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.
+* {op-system} now supports boot disk mirroring, except on s390x, providing redundancy in the case of disk failure.
+* {op-system} on s390x can be installed onto FBA DASD disks.
+
+[NOTE]
+====
+LUKS configuration can no longer be performed using `/etc/clevis.json`. Provisioning will fail if `/etc/clevis.json` is specified in the machine config.
+====
+
+[id="ocp-4-7-bootupd"]
+==== Update the bootloader using bootupd
+
+With bootupd, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
+
+[id="ocp-4-7-rhcos-rhel-8-3-packages"]
+==== {op-system} now supports RHEL 8.3
+
+{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This is relevant for hardware support, new NetworkManager features, and so on.
```

Comment:
```suggestion
{op-system} now uses Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will continue to use RHEL 8.2 packages. This is relevant for hardware support, new NetworkManager features, and other version-specific items.
```

> ! Use _and so on_ when you list a clear sequence of elements, such as “1, 2, 3, and so on” or “Monday, Tuesday, Wednesday, and so on.” Otherwise, rewrite the sentence to replace etc. with something more descriptive, such as “and other output.”

Reply:
Thanks, I didn't like "and so on" so I've gotten a bit more descriptive here:

> This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.